cmd/podman/main_local.go

```diff
@@ -243,6 +243,22 @@ func setupRootless(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func useUserNS(cmd *cobra.Command) bool {
+	if os.Geteuid() == 0 {
+		return false
+	}
+	switch cmd {
+	case _migrateCommand,
+		_mountCommand,
+		_renumberCommand,
+		_infoCommand,
+		_searchCommand,
+		_versionCommand:
+		return false
+	}
+	return true
+}
```

Comment:
I'd like a better function name and/or a function comment over what this function is supposed to do.

Reply:
Suggestions?

Reply:
Changed name to executeCommandInUserNS
and added a comment.